cypress/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Could we do like other modules and have something like ` "extends": "../../tsconfig-base",`